build.gradle

```diff
@@ -103,6 +100,26 @@ subprojects {
 		archivesBaseName = "${project.name}"
 	}
 
+	// Support for some higher language versions can only be achieved using Toolchains.
+	// Compatibility matrix at https://docs.gradle.org/current/userguide/compatibility.html#java
+	// See https://docs.gradle.org/current/userguide/toolchains.html#toolchains
+	java {
+		toolchain {
+			languageVersion = JavaLanguageVersion.of(11)
+		}
+	}
+
+	// This ensures bytecode compatibility with Java 8 by ensuring that symbols not
+	// available in Java 8 are not used in the source code.
+	// This is equivalent of setting --release flag in Java compiler introduced in Java 9.
+	// Note that the toolchain used is Java 11 - which means Java 11 is required
+	// to build this project.
+	afterEvaluate {
```

Comment:
nit does this need to be in `afterEvaluate`? https://github.com/search?type=code&q=options.release+%3D+8+lang%3Agradle

Reply:
No, it looks like `afterEvaluate` is extraneous here.